package.json

```diff
@@ -51,7 +51,9 @@
         "express-jwt-permissions": "^1.3.6",
         "express-rate-limit": "^5.3.0",
         "express-validator": "^6.12.1",
+        "file-type": "^14.6.2",
         "got": "^11.8.5",
+        "install": "^0.13.0",
```

Comment:
@valeriagrazzini For this `install` package I also can really find a reason why it should be added (based this only on this diff btw, you probably have a reason)